src/framework/components/particlesystem/particlesystem_system.js

```diff
@@ -197,10 +197,35 @@ pc.extend(pc, function() {
             }, {
                 name: "spawnBounds",
                 displayName: "Spawn Bounds",
-                description: "Defines an AABB in which particles are allowed to spawn",
+                description: "Defines an AABB/radius in which particles are allowed to spawn",
```

Comment:
How would a vector represent a sphere radius? Do we need to create separate properties for box and sphere bounds? The UI will simply hide/show these based on the setting of spawnShape.